src/app/campaign-list/campaign-list.service.ts

```diff
@@ -29,6 +29,15 @@ export interface CampaignParams {
   desc?: boolean;
 }
 
+// TODO: Omit is added in TS 3.5
```

Comment:
[Omit](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittk) comes in TypeScript 3.5. I tried to install it, but Angular 7 isn't ready for it.
Omit is useful so I can show the association between the CampaignParams and the CampaignRouteParams interfaces; the properties that are overridden in route params are string interpretations of the values.